libs/shared/sai-editor/src/sai-editor.component.html

```diff
@@ -557,8 +557,15 @@
         <div class="col-12">
           <keira-editor-buttons [editorService]="editorService"></keira-editor-buttons>
 
-          <button id="generate-comments-btn" class="btn btn-primary btn-sm float-end" (click)="editorService.generateComments()">
-            Generate comments
+          <button id="generate-comments-btn" class="btn btn-primary btn-sm float-end" (click)="editorService.generateComments(true)">
+            Generate comments (all)
+          </button>
+          <button
+            id="generate-comments-btn-single"
+            class="btn btn-primary btn-sm float-end me-2"
+            (click)="editorService.generateComments()"
+          >
+            Generate comment
```

Comment:
shouldn't we use translations? also the text should be more specific, like "Generate comment for selected row"

Reply:
also this button should be disabled if no row is selected